terraform/modules/quorum-cluster-region/bootnodes.tf

```diff
@@ -32,66 +32,103 @@ resource "aws_subnet" "bootnodes" {
 
-  provisioner "remote-exec" {
-    inline = [
-      "sudo apt-get -y update",
-      "echo '${aws_s3_bucket.quorum_constellation.id} /opt/quorum/constellation/private/s3fs fuse.s3fs _netdev,allow_other,iam_role 0 0' | sudo tee /etc/fstab",
-      "sudo mount -a",
-      "echo '${count.index}' | sudo tee /opt/quorum/info/index.txt",
-      "echo '${jsonencode(var.bootnode_counts)}' | sudo tee /opt/quorum/info/bootnode-counts.json",
-      "sudo python /opt/quorum/bin/fill-node-counts.py --quorum-info-root '/opt/quorum/info' --bootnode",
-      "echo '${var.aws_region}' | sudo tee /opt/quorum/info/aws-region.txt",
-      "echo '${var.primary_region}' | sudo tee /opt/quorum/info/primary-region.txt",
-      # This should be last because init scripts wait for this file to determine terraform is done provisioning
-      "echo '${var.network_id}' | sudo tee /opt/quorum/info/network-id.txt",
-    ]
+  launch_configuration = "${element(aws_launch_configuration.bootnode.*.name, count.index)}"
+
+  min_size = 1
+  max_size = 1
+  desired_capacity = 1
+
+  health_check_grace_period = 300
+  health_check_type = "ELB"
+
+  vpc_zone_identifier = ["${element(aws_subnet.bootnodes.*.id, count.index)}"]
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# BOOTNODE LOAD BALANCER
+# ---------------------------------------------------------------------------------------------------------------------
+resource "aws_lb" "bootnodes" {
+  count = "${aws_launch_configuration.bootnodes.count}"
+
+  internal = false
+  load_balancer_type = "network"
+  subnets = ["${aws_subnet.bootnodes.*.id}"]
+
+  # Q?: Apparently load balancers can't be modified without deleting them(?)
+  # Examples set this to true even though default is false, not sure why 
+  # enable_deletion_protection = true
+}
+
+resource "aws_lb_listener" "bootnode" {
+  count = "${aws_launch_configuration.bootnodes.count}"
+  load_balancer_arn = "${aws_lb.bootnodes.*.arn}"
+  protocol = "TCP"
+  port = "30301"
+
+  default_action {
+    target_group_arn = "${aws_lb_target_group.bootnodes.*.arn}"
+    type = "forward"
   }
 }
 
+# TODO: enodes need to be accessible via both TCP & UDP.
+# Where do I configure the UDP access here?
+
+resource "aws_lb_target_group" "bootnodes" {
+  count = "${aws_launch_configuration.bootnodes.count}"
+  arn = "${aws_autoscaling_group.bootnodes.*.arn}"
+  port = "30301"
+}
+
 # ---------------------------------------------------------------------------------------------------------------------
 # THE USER DATA SCRIPT THAT WILL RUN ON EACH BOOTNODE WHEN IT'S BOOTING
 # This script will configure and start the Consul Agent
 # ---------------------------------------------------------------------------------------------------------------------
 
 data "template_file" "user_data_bootnode" {
+
+  # Q?: This looks like it assumes only one bootnode per region, is that true?
```

Comment:
No it does not assume that, but it currently uses a single user-data for all bootnodes, and any difference between them comes from the terraform provider.

I think you will need to turn this into multiple user-data.